wangle/CMakeLists.txt

```diff
@@ -84,7 +84,7 @@ target_link_libraries(wangle
   ${OPENSSL_LIBRARIES}
   ${GLOG_LIBRARY_PATH}
   ${GFLAGS_LIBRARY_PATH}
-  -latomic)
+  $<$<NOT:$<CXX_COMPILER_ID:Clang>>:latomic>)
```

Comment:
you're right, I've accidentally removed the hyphen while making this parameter absent with clang. I've updated the pull request with a fix.